app/controllers/ops_controller/settings/cap_and_u.rb

```diff
@@ -28,12 +28,8 @@ def cu_collection_update
       end
 
       unless @edit[:new][:storages] == @edit[:current][:storages] # Check for storage changes
-        @st_recs = Storage.all.each_with_object({}) { |st, h| h[st.id] = st }
-        @edit[:new][:storages].each_with_index do |s, si|
-          if s[:capture] != @edit[:current][:storages][si][:capture]
-            ds = @st_recs[s[:id]]
-            ds.perf_capture_enabled = s[:capture] if ds
-          end
+        @edit[:new][:storages].each do |_, s|
+          s[:st_rec].perf_capture_enabled = s[:capture]
```

Comment:
we may want to keep the old mechanism of storing just the id in session and using that to load the real object, right?

Reply:
I wanted to clean it up in a future PR, but I can do it in this one.